src/theme.js

```diff
@@ -43,7 +43,7 @@ const colorTheme = createTheme({
 const theme = createTheme(colorTheme, {
   typography: {
     fontFamily: ['Lato', 'Roboto', 'Helvetica', 'Arial', 'sans-serif'].join(
-      ',',
+      ',',letterSpacing: 0,
     ),
     fontWeight: 400,
```

Comment:
```suggestion
      ','
    ),
    letterSpacing: 0,
    fontWeight: 400,
```

I think you meant to do this. You can use `npm run lint` to detect these errors.